.github/workflows/tag-release.yml

```diff
@@ -8,12 +8,12 @@ on:
       commit-ref:
         description: 'Branch or SHA to tag (defaults to master)'
         required: false
-  schedule:
-    - cron: '0 5 * * 1-5' # Run every weekday 5:00 UTC
+  workflow_call:
```

Comment:
I may be wrong, but I'm not sure this is needed 🤔

Reply:
You are wrong 😀 https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call
The `workflow_dispatch` event is for manually triggering the workflow

Reply:
aha! TIL

Reply:
Me too!